pygmt/base_plotting.py

```diff
@@ -634,7 +643,7 @@ def basemap(self, **kwargs):
         [optionally] gridlines. A simple map scale or directional rose may also
         be plotted.
 
-        At least one of the options *B*, *L*, or *T* must be specified.
+        At least one of the options *frame*, *L*, or *T* must be specified.
```

Comment:
I've only changed `B` to frame here. GMT.jl uses map_scale for `L`, rose for `Td` and compass for `Tm` (see https://www.generic-mapping-tools.org/GMT.jl/latest/#GMT.basemap), do we want to go with those?

Reply:
Yes to me.